.licenserc.json

```diff
@@ -0,0 +1,74 @@
+{
+    "**/*.{go,cs,h,cpp,js,ts,php,rs}": [
+      "// Copyright The OpenTelemetry Authors",
+      "//",
+      "// Licensed under the Apache License, Version 2.0 (the \"License\");",
+      "// you may not use this file except in compliance with the License.",
+      "// You may obtain a copy of the License at",
+      "//",
+      "//     http://www.apache.org/licenses/LICENSE-2.0",
+      "//",
+      "// Unless required by applicable law or agreed to in writing, software",
+      "// distributed under the License is distributed on an \"AS IS\" BASIS,",
+      "// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.",
+      "// See the License for the specific language governing permissions and",
+      "// limitations under the License."
```

Comment:
in case you hadn't seen this, we can also use the short form:

> Copyright The OpenTelemetry Authors
> SPDX-License-Identifier: Apache-2.0

https://github.com/open-telemetry/community/blob/main/CONTRIBUTING.md#code-attribution

Reply:
I hadn't, good catch. Let's save those bytes.